src/Draggable/README.md

```diff
@@ -89,7 +89,7 @@ Sensors dictate how drag operations get triggered, by listening to native browse
 By default draggable includes the `MouseSensor` & `TouchSensor`. Default: `[]`
 
 **`appendTo {String|HTMLElement|Function}`**  
-Draggable allows you to specify where the mirror should be appended to. You can specify a css
+Draggable allows you to specify where the mirror should be appended to.  For clarification, this is not where the destination will be placed, only the temporary mirror object (the object under your cursor as you drag). You can specify a css
```

Comment:
👍

I'd change it up a little: `For clarification, this is not where the source will be placed, only the temporary mirror element, which is the element that follows your cursor as you drag.`

Reply:
Also, could you remove the new line after `You can specify a css...`?

Reply:
cc @ianmart1n

Reply:
sure!